main.go

```diff
@@ -49,10 +51,23 @@ func mkconf() Config {
 		}
 	}
 
-	dbstring, exists := os.LookupEnv("GODRINK_DB")
-	if !exists {
-		log.Fatal("No database given, exiting!")
+	dbdriver, driverExists := os.LookupEnv("GODRINK_DBDRIVER")
+	dbstring, connExists := os.LookupEnv("GODRINK_DB")
```

Comment:
While fine, the variable connExists threw me for a loop because I kept thinking it meant the connection was already established. Maybe something like `dbUrl, dbUrlExists` would be even better?

Reply:
I agree, changed the names to your suggestion